overlay.d/15fcos/usr/lib/systemd/system/zincati.service.d/010-oci-migration.conf

```
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=/usr/libexec/coreos-oci-rebase
```

```suggestion
ExecStartPre=-/usr/libexec/coreos-oci-rebase
```

I wonder if we should not care about the exit code of the rebase script and then we can change the `exit 0` in the script where we are actually having a failure into `exit 1` now and `systemctl status zincati` will indicate some problem as well, although it won't be fatal.